website/source/docs/agent/options.html.md

```diff
@@ -717,6 +717,14 @@ Consul will not enable TLS for the HTTP API unless the `https` port has been ass
         `write` access to this backend, as well as permission to mount the backend at this path if it is not 
         already mounted.
 
+        #### Common CA Config Options
+
+        <p>There are also a number of common configuration options supported by all providers:</p>
+        
+        * <a name="ca_leaf_cert_ttl"></a><a href="#ca_leaf_cert_ttl">`leaf_cert_ttl`</a> The lease duration of
+        a leaf certificate issued for a service, after which a new certificate will be requested by the proxy.
```

Comment:
Suggest we make this more precise. This is the _hard_ limit on certificate validity, we actually request a new one with time to spare. We should probably also point out the trade off here that this is an upper bound on how long a control plane outage the cluster can handle before network connections will start being rejected which is why it's 3 days as a default to survive a weekend outage without intervention.